### DISCUSSION TITLE
Mysterious `Heart On Your Sleeve` badge

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#30] [Hunt Log]
There is a lot of interest in the community about how to earn the `Heart On Your Sleeve` and the `Open Sourcerer` badges. For the past year, not a single user has been seen with one of these badges on the GitHub platform. Which begs the question: Did they ever really exist?

Well, some sources reported that both achievements are currently not publicly available on GitHub. And we will have to wait until for the official launch. Quote from @melvincwng - [source #1][1] and from @msuliq - [source #2][2].
 
A few weeks ago, @vnothkumar spotted this [YouTube short](https://www.youtube.com/shorts/1UgqjSCGslg) with @damovisa (GitHub staff member) explaining new GitHub features, while his profile shows `x2 Heart On Your Sleeve` and `x3 Open Sourcerer` badges. But, they no longer appear on Damian's most recent profile. This proves the point: They are still in the experimental stage.

Don't miss out! Keep an eye on the official [GitHub announcements](https://github.blog) for the latest.

![image](https://user-images.githubusercontent.com/5120290/219281456-e53a1e3d-a484-4ef4-bad5-9a475819ea19.png)

[1]: https://github.com/orgs/community/discussions/43514#discussioncomment-4618898
[2]: https://github.com/orgs/community/discussions/44972#discussioncomment-4744483
[3]: https://github.com/community/community/discussions/41863#discussioncomment-4896147

**Comment by ParanoidUser** [comment]
If you find this information useful, please `mark as answer` this comment. Thanks!

**Reply by ViktorSvertoka** [reply]
Hello!
I found this information, I do not know how true it is ...
![Знімок екрана 2023-05-07 о 14 05 50](https://user-images.githubusercontent.com/115661003/236673840-674c6b97-1031-425d-84c9-6a3ac4bfe3d7.png)

**Reply by satyam-umrao** [reply]
not released yet 
!

**Comment by ghost** [comment]
you probably getting a galaxy brain🌀

**Reply by ParanoidUser** [reply]
> you probably getting a galaxy brain🌀

Hey @vnothkumar, thanks for jumping in! 🤞

**Comment by Tynab** [comment]
To earn the Open Sourcerer achievement on GitHub, you need to make a qualifying contribution to an open source project on GitHub. Here are the steps you can follow:
1. Sign up for a GitHub account if you haven't already.
2. Find an open source project on GitHub that you would like to contribute to. You can search for open source projects using GitHub's Explore feature, or you can look for projects in specific programming languages or topics that interest you.
3. Fork the repository of the open source project that you want to contribute to. This creates a copy of the project in your own GitHub account that you can make changes to.
4. Make a qualifying contribution to the project. This can be anything from fixing a bug or adding a new feature, to improving documentation or submitting a new translation. Your contribution must be accepted and merged into the main repository by the project maintainers to count towards the Open Sourcerer achievement.
5. Once your contribution has been merged, you should receive a notification from GitHub that your contribution has been accepted. After 30 days of making your first contribution, you will earn the Open Sourcerer badge on your GitHub profile.
Note that GitHub has specific criteria for what constitutes a qualifying contribution, so be sure to review the contribution guidelines for the project you want to contribute to before getting started. Good luck!

**Reply by thenithinbalaji** [reply]
Can you give the link to profiles that have these badges?

**Comment by ParanoidUser** [comment]
Hey @Tynab, thank you for chiming in!

This is very enlightening 🤩. As I understand it, to earn:
  - `Heart on Your Sleeve` create a project with open source license (like CC0 or MIT);
  - `Open Sourcerer` contribute to open-source project (like Apache, Spring, etc.);

Would you like to link back to the insights source?
